### ISSUE TITLE
Tested on STM32F411CEUx

### ISSUE THREAD

**Issue by PanRe** [dmitrystu/libusb_stm32#47]
Hi Dmitry,

Just wanted to let you know your stack also works for STM32F411xE devices. I just had to adapt your usb.h with 
```c
#elif defined(STM32F411xE)

#define USBD_STM32F429FS
extern const struct usbd_driver usbd_otgfs;
#define usbd_hw usbd_otgfs
```

to let everything work. I tested it with your CDC and HID demo.

By the way i wanted to ask you if you plan on publishing also the mass storage device .h files (as i read in other issues, you mentioned there are some parts already done). Further, i want to implement an audio device but before starting on my own, is there anything like that already available? Ah and are you planning on doing some host or even OTG stacks too?

Thank you and thanks for this very nice light weight stack!
